webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/RuleGrid.scala

```diff
@@ -290,7 +276,7 @@ class RuleGrid(
         val onLoad              = {
           s"""createRuleTable (
                   "${htmlId_rulesGridId}"
-                , ${tableData.json.toJsCmd}
+                , ${tableData}
```

Comment:
This breaking change remains, but I think this is the last one : 
```suggestion
                , ${tableData.toJson.toJsCmd}
```